tmpl/blog/2017-winter-hackathon-roundup.md

```diff
@@ -0,0 +1,248 @@
+# MirageOS Winter Hack Retreat, Marrakesh 2017
+
+This winter, 33 people from around the world gathered in Marrakesh for a Mirage hack retreat. This is fast becoming a [MirageOS](/blog/2016-spring-hackathon) [tradition](/blog/2017-march-hackathon-roundup), and we're a little sad that it's over already! We've collected some trip reports from those who attended the 2017 winter hack retreat, and we'd like to thank our amazing hosts, organisers and everyone who took the time to write up their experiences.
+<img src="/graphics/winter2017.jpg" style="float:right; padding: 15px" />
+
+We, the MirageOS community, strongly beliefs in using their own software: this website is a unikernel since day one^W^W it is possible to run MirageOS unikernels.  In Marrakesh we used our own DHCP and DNS server without trouble.  There are many more services under heavy development (including git, ssh, ...), which we're looking forward to use soon ourselves.
```

Comment:
Some agreement nitpicking: "We, the MirageOS community, strongly believe in using our own software: this website has been a unikernel since day one ^W^W it has been possible to run MirageOS unikernels."  and ", which we're looking forward to using soon ourselves".